go/libraries/doltcore/diff/diff_stat.go

```diff
@@ -131,7 +131,10 @@ func diffProllyTrees(ctx context.Context, ch chan DiffStatProgress, keyless bool
 		}
 	}
 
-	err = prolly.DiffMaps(ctx, f, t, func(ctx context.Context, diff tree.Diff) error {
+	// TODO: Use `vMapping` to determine whether or not the schema has changed. If it has, then all rows should
+	// count as modifications in the diff.
```

Comment:
(super minor) I was thinking about this some more... probably more accurate to say "to determine whether columns have been added or removed" instead of "schema has changed". That's what `vMapping` is already tracking of course, but a little more accurate since for other schema changes, like changing a column's type, we wouldn't want to show all rows as modified. 

```suggestion
	// TODO: Use `vMapping` to determine whether columns have been added or removed. If so, then all rows should
	// count as modifications in the diff.
```